migrations/20220201085056_pulling_data/migration.sql

```diff
@@ -1 +0,0 @@
--- This is an empty migration.
```

Comment:
Please undo the deletion of this migration, even though it's empty its part of the migration history which needs it